### ISSUE TITLE
[Request]

### ISSUE THREAD

**Issue by donutholer** [jiaweihli/manga_downloader#94]
Can someone show me how to use this in a script? If im on mac I just run it from terminal?

**Comment by CharlieCorner** [comment]
This is indeed a command line script, you need to use Python 2 in order to use it, as for the usage [you can check the usage section on the homescreen for the project, it contains examples and documentation on the various options](https://github.com/jiaweihli/manga_downloader#usage).

**Comment by donutholer** [comment]
Yep I found out how to use it. I'm having troubles changing the download
directory. While i'm in inside ' src' in the terminal, do I type
'--directory<drag the directory on to terminal>' to change it? Because that
does not seem to work. (Mac user) Thanks for the fast response and creating
this script!

**Comment by CharlieCorner** [comment]
You need to provide an absolute path. Be also sure to leave a space between the option and the value
`--directory /path/to/your/directory`
